manifests/modelserving/servingruntimes-template.yaml

```diff
@@ -0,0 +1,56 @@
+kind: Template
```

Comment:
I don't see the old servingruntimes-config, as used in odh-manifests, anywhere in odh-dashboard. 
[odh-manifests/blob/master/odh-dashboard/modelserving/servingruntimes-config.yaml](https://github.com/opendatahub-io/odh-manifests/blob/master/odh-dashboard/modelserving/servingruntimes-config.yaml) Will that configmap be deprecated or removed in favor of this template approach when the next odh-manifests are built?

Reply:
Yes, you can check all the features added [in this new release](https://github.com/opendatahub-io/odh-dashboard/issues/1069), but the configmap will be deprecated as it was a workaround until we implemented this. Users will have a section to update and add their custom runtimes, and they will be able to choose between all of them.